.github/workflows/static-analysis.yml

```
@@ -0,0 +1,75 @@
+              
+              # Commit and push changes
+              git add .
+              git commit -m "chore: auto-fix linting and formatting issues"
+              git push
+            fi
+          fi
+        fi
+    
+    - name: Generate ESLint report
+      run: npx eslint "{src,apps,libs,test}/**/*.ts" --format json --output-file eslint-report.json || true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)